stdlib/public/core/StringBridge.swift

```diff
@@ -413,10 +413,40 @@ private func _getCocoaStringPointer(
   return .none
 }
 
+#if arch(arm64)
+//11000000..payload..111
+private let constantTagMask:UInt =
+  0b1111_1111_1000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0111
+private let expectedConstantTagValue:UInt =
+  0b1100_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0111
+#endif
```

Comment:
Do a `var` instead so it doesn't have storage (it can be inlined)

Reply:
wait lets can't be inlined? That's wacky. Will do

Reply:
I'm not positive on this, they might be able to be and eliminated at the LLVM level, but computed vars are the closest to macros in Swift